openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -0,0 +1,240 @@
+"""
+Declaration of CourseOverview model
+"""
+
+from datetime import datetime as date_time  # the as clause avoids naming conflicts the DateTime field
+import json
+from util.date_utils import strftime_localized
```

Comment:
move this date_utils import after the block of django-related imports since date_utils is an edx-platform module.